ocsp/responder/db_source.go

```diff
@@ -62,11 +61,9 @@ func (src *dbSource) Response(ctx context.Context, req *ocsp.Request) (*Response
 	}
 
 	if certStatus.IsExpired {
-		src.log.Infof("OCSP Response not sent (expired) for CA=%s, Serial=%s", hex.EncodeToString(req.IssuerKeyHash), serialString)
 		src.counter.WithLabelValues("expired").Inc()
 		return nil, ErrNotFound
```

Comment:
Do we want to incorporate the more details expired vs zero-lastUpdated messaging into an error that wraps ErrNotFound? Wrapping should be fine as long as we're checking for it using errors.Is().